JayJayson84/VSIX.Umbraco.DictionaryBuilder
# Update icons for better clarity in IDE themes

# Description

Update project icons for better clarity in IDE themes.

### Type of change

Tick the relevant option.


## Where should the reviewer start?

- Clone the branch.
- Run the project in Visual Studio.

## How should this be tested?

1. Set the IDE colour theme to different values in **Options > Environment > General**.
2. Right click on a project in the Solution Explorer and check the icons in the **Umbraco Dictionary** sub menus.
